test/io/testscripts.py

```diff
@@ -103,14 +103,17 @@ def test_main_empty_argv(self):
         with self.assertRaises(SystemExit):
             cclib.scripts.ccframe.main()
 
+    @unittest.skipIf(sys.version_info < (3, 2), "assertRaisesRegex is Python >= 3.2")
```

Comment:
I'm a bit concerned that this won't ever get tested with Python 2.7+. I'd be fine with letting it slide if there wasn't an elegant solution, but it looks like  just plain `assertRaises` could be used and the exception message checked afterwards. See the example in the documentation https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertRaises.

Reply:
Ok, I misread the documentation and didn't know that `assertRaisesRegexp` works on all versions.